custom/icds_reports/static/js/directives/service-delivery-dashboard/service-delivery-dashboard.directive.js

```diff
@@ -20,7 +20,7 @@ function ServiceDeliveryDashboardController($scope, $http, $location, $routePara
 
     vm.dtOptions = DTOptionsBuilder.newOptions()
         .withOption('ajax', {
-            url: url('service_delivery_dashboard'),
+            url: url('service_delivery_dashboard', vm.step),
```

Comment:
where do we actually set the value of this step variable?

Reply:
we set this value 3 lines above  (line 19).